download.py

```diff
@@ -846,9 +846,14 @@ def download(asset_data, **kwargs):
         asset_data = copy.deepcopy(asset_data)
     else:
         asset_data = asset_data.to_dict()
+
+    #inject resolution into prefs.
+    PREFS = utils.get_preferences_as_dict()
+    PREFS["resolution"] = kwargs.get("resolution", "original")
```

Comment:
This will download correct HDRI? 😮

Reply:
Yes, indeed. The client gets the resoution that should be downloded from preferences.